test/jdk/java/nio/channels/DatagramChannel/Promiscuous.java

```diff
@@ -222,9 +208,8 @@ public static void main(String[] args) throws IOException {
             InetAddress source = config.ip4Addresses(nif).iterator().next();
             test(INET, nif, ip4Group1, ip4Group2);
 
-            // Solaris and Linux allow IPv6 sockets join IPv4 multicast groups
-            if (os.equals("Linux"))
-                test(UNSPEC, nif, ip4Group1, ip4Group2);
+            // test IPv6 sockets joining IPv4 multicast groups
+            test(UNSPEC, nif, ip4Group1, ip4Group2);
```

Comment:
The SAP and/or IBM folks that maintain the AIX port will probably have to adjust this to skip on that platform.